Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -27,12 +27,14 @@ public class NetworkClient : MonoBehaviour, INetworkClient
     {
         static readonly ILogger logger = LogFactory.GetLogger(typeof(NetworkClient));
 
-        [Header("Authentication")]
```

Comment:
Any reason for the removal of this header?

Reply:
Only one field was related to authentication and the rest of the fields were under it. Doesn't make sense.

Reply:
I dont mind the change. The way the inspector looks is nice and clean:
![image](https://user-images.githubusercontent.com/1385108/107536082-6392ea00-6b8f-11eb-9873-a1be138adc8d.png)